README.md

```diff
@@ -1290,26 +1291,111 @@ The output of the script consists of JSON log lines.
 
 One log line is output for each missing object,
 
-```
-{"name":"s3utils:verifyReplication","time":1670279729106,"key":"rna-4/rna33","size":0,"srcLastModified":"2022-12-01T12:27:11.469Z","level":"info","message":"object missing in destination","hostname":"scality.local","pid":24601}
+```json
+{
+  "name":"s3utils:verifyReplication",
+  "time":1670279729106,
+  "key":"rna-4/rna33",
+  "size":0,"srcLastModified":"2022-12-01T12:27:11.469Z",
+  "level":"info",
+  "message":"object missing in destination",
+  "hostname":"scality.local",
+  "pid":24601
+}
 ```
 
 and for each size mismatched object,
 
+```json
+{
+  "name":"s3utils:verifyReplication",
+  "time":1670279723474,
+  "key":"rna-2/34",
+  "srcSize":1,
+  "dstSize":0,
+  "srcLastModified":"2022-12-01T11:31:27.528Z",
+  "dstLastModified":"2022-12-01T11:33:42.000Z",
+  "level":"info",
+  "message":"object size does not match in destination",
+  "hostname":"scality.local",
+  "pid":24601
+}
 ```
-{"name":"s3utils:verifyReplication","time":1670279723474,"key":"rna-2/34","srcSize":1,"dstSize":0,"srcLastModified":"2022-12-01T11:31:27.528Z","dstLastModified":"2022-12-01T11:33:42.000Z","level":"info","message":"object size does not match in destination","hostname":"scality.local","pid":24601}
+
+and for each failed metadata retrieval (after retries),
+
+```json
+{
+  "name":"s3utils:verifyReplication",
+  "time":1670887351694,
+  "error":{
+    "message":"socket hang up",
+    "code":"TimeoutError",
+    "time":"2022-12-12T23:22:31.694Z",
+    "region":"us-east-1",
+    "hostname":"localhost",
+    "retryable":true
+    },
+  "bucket":"src-bucket",
+  "key":"pref1/key404",
+  "level":"error",
+  "message":"error getting metadata",
+  "hostname":"scality.local",
+  "pid":11398
+}
 ```
 
 The script also logs a progress update (a summary), every 10 seconds by default,
```

Comment:
The following JSON example does not look related to progress update, but to final completion of the script.